### ISSUE TITLE
Diff target 'http://<PATH>' was not found in the repository at revisions '0' and '1'

### ISSUE THREAD

**Issue by moelski** [hartwork/svneverever#74]
Hi !
Just try svneverever because I need to migrate a SVN repo to git. 
The one repo consists of several projects in different pathes.

When I try to checkout a specific path I got this error: 
```
Analyzing 39397 revisions...
  0.00%  (00:00:07 remaining)  [                                                                                                                                                                                               ]Traceback (most recent call last):
  File "/home/user/.local/bin/svneverever", line 8, in <module>
    sys.exit(main())
  File "/home/user/.local/lib/python3.10/site-packages/svneverever/__main__.py", line 370, in main
    summary = client.diff_summarize(
pysvn._pysvn.ClientError: Diff target 'http://svnserver:82/svn/ZW/CAN2x' was not found in the repository at revisions '0' and '1'
```

Any chance to fix this?

**Comment by hartwork** [comment]
Hi @moelski,

could you try if one of the known intact subversion repositories works in your environment? To be concrete, please try this:
```bash
cd "$(mktemp -d)"
python3.10 -m venv venv --system-site-packages
source venv/bin/activate
pip install svneverever
svnadmin create bs2b_svn_store
time sh -c 'svnrdump dump svn://svn.code.sf.net/p/bs2b/code/ | svnadmin load bs2b_svn_store/'
svneverever bs2b_svn_store/
```

That will let us know if this behavior happens for you with all repositories or if it's in some way particular to your `http://svnserver:82` case. Thanks!

**Comment by hartwork** [comment]
@moelski quick update, I can reproduce it now:

```
# svneverever svn://svn.code.sf.net/p/bs2b/code/tags/plugins/xmms/0.4.0
Analyzing 50 revisions...
  2.00%  (00:00:01 remaining)  [##                                                                                                                                     ]Traceback (most recent call last):
  File "/tmp/tmp.B4bcmAZFeB/venv/bin/svneverever", line 8, in <module>
    sys.exit(main())
  File "/tmp/tmp.B4bcmAZFeB/venv/lib/python3.10/site-packages/svneverever/__main__.py", line 370, in main
    summary = client.diff_summarize(
pysvn._pysvn_3_11.ClientError: Diff target 'svn://svn.code.sf.net/p/bs2b/code/tags/plugins/xmms/0.4.0' was not found in the repository at revisions '0' and '1'
```

The short answer is: svneverever always needs the full repository. I can add code to handle this exception and produce nice error message instead, but your current usage is not supported. To make things work for you, please use svneverever as described at https://github.com/hartwork/svneverever#usage using a remote dump of the full repository. Thanks!

**Comment by hartwork** [comment]
@moelski released svneverever [1.7.2](https://github.com/hartwork/svneverever/releases/tag/v1.7.2) to GitHub and PyPI now which no longer shows this backtrace but an error message.
